packages/jaeger-ui/src/utils/readJsonFile.tsx

```diff
@@ -22,8 +22,10 @@ export default function readJsonFile(fileList: { file: File }) {
       }
       try {
         resolve(JSON.parse(reader.result));
-      } catch (error: any) {
-        reject(new Error(`Error parsing JSON: ${error.message}`));
+      } catch (error: unknown) {
```

Comment:
can we not just do this?
```suggestion
      } catch (error: Error) {
```

Reply:
I already try to use `Error` instead `unknown` but It gives `Catch clause variable type annotation must be 'any' or 'unknown' if specified.` error.